backend/src/index.ts

```diff
@@ -85,6 +88,14 @@ io.on('connection', (socket: ChatSocket) => {
     );
   });
 
+  socket.on('reaction', (reactionId) => {
+    console.log(`user ${socket.userId} reaction ${reactionId}`);
+    var reactionMap =  reactions[reactionId] || {};
+    reactionMap[socket.userId] = new Date();
+    reactions[reactionId] = reactionMap;
+    io.in(socket.sessionId).emit('reactions', reactions);
```

Comment:
`reactions` powinno być "session aware" czyli to pownien być obiekt idik sesji -> reakcje